### ISSUE TITLE
PendingIntent seems only getBroadcast

### ISSUE THREAD

**Issue by narychen** [Neson/react-native-system-notification#3]
Clicking the notification does not start the main activity.
Is there any way to do it?

**Comment by avishayil** [comment]
+1 on this

**Comment by Neson** [comment]
This bug seems to occur on RN 0.17 since I've done the development on 0.16 and haven't test it on the new version for now. I'm currently figuring it out :smiley: 

Are you guys all using 0.17?

**Comment by joacodeviaje** [comment]
I am.

**Comment by avishayil** [comment]
Me too

**Comment by Neson** [comment]
And what are your Android versions by the way?

**Comment by avishayil** [comment]
5.1.1

**Comment by Neson** [comment]
I just tested this using https://github.com/Neson/ReactNativeSystemNotificationDev and it turns out that the problem is related to Android version - on 5.1.0 it works fine, but not 4.1.1.

Heading to figure it out and probably solve this with #2.

**Comment by Neson** [comment]
2431ec6 might have fixed this, and it's published to NPM with the version 0.0.8 (tested on API level 16 (4.1.1), 21 (5.0.0) and 22 (5.1.0)).You can try and let me know if it works :smile:.

Note that a new permission needs to be add to `android/app/src/main/AndroidManifest.xml`: 

```xml
    <uses-permission android:name="android.permission.GET_TASKS" />
```
